pkg/cache/configmaps.go

```diff
@@ -1,10 +1,11 @@
 package cache
 
 import (
-	sharev1alpha1 "github.com/openshift/csi-driver-projected-resource/pkg/api/projectedresource/v1alpha1"
-	"k8s.io/klog/v2"
 	"sync"
 
+	sharev1alpha1 "github.com/openshift/csi-driver-shared-resource/pkg/api/projectedresource/v1alpha1"
```

Comment:
```suggestion
	sharev1alpha1 "github.com/openshift/csi-driver-shared-resource/pkg/api/sharedresource/v1alpha1"
```

Reply:
This PR doesn't have the api rename, only the module rename. I think this will get tossed out when the changes from openshift/api get merged in.